miden-lib/src/transaction/inputs.rs

```diff
@@ -265,11 +265,17 @@ fn add_account_to_advice_inputs(
 /// The advice provider is populated with:
 ///
 /// - For each note:
-///     - The note's authentication path against its block's note tree.
-///     - The note's input padded prefixed by its length.
-///     - The note's asset padded.
-/// - And all notes details together under the input notes' commitment.
+///     - The note's details (serial number, script root, and its' input / assets hash).
+///     - The note's private arguments.
+///     - The note's public metadata.
+///     - The note's public inputs data. Prefixed by its length and padded to an even word length.
+///     - The note's asset padded. Prefixed by its length and padded to an even word length.
+///     - For autheticated notes:
```

Comment:
Should we also mention `is_authenticated` flag here?

Reply:
added